### DISCUSSION TITLE
Getting Kubernetes Events

### DISCUSSION THREAD

**Discussion by allahshukur-ahmadzada** [fluent/fluent-bit#6942] [Q&A]
I wonder can fluentbit fetch and parse kubernetes events? if yes please share doc section for it.

**Comment by kyleli666** [comment]
There is a related input plugin https://docs.fluentbit.io/manual/pipeline/inputs/kubernetes-events

But I'm wondering if it'll get duplicated events with a DaemonSet installation?

**Reply by patrick-stephens** [reply]
You will currently yes, you should use a deployment in this case - there's no need to put everything in the DS and it limits blast radius too.
There are some feature requests around this on implementing leader-election support and the like to handle this properly though.

**Reply by koakenred** [reply]
Is still the case that you will get duplicated events with a DaemonSet installation?

**Reply by patrick-stephens** [reply]
Yes, I don't believe any implementation of leader-election or similar is in place.
